schemas/iso19139/src/main/plugin/iso19139/layout/layout-custom-fields.xsl

```diff
@@ -137,4 +137,106 @@
       </xsl:with-param>
     </xsl:call-template>
   </xsl:template>
+
+
+  <!-- SDS: Category. Render an anchor as select box populating it with a codelist -->
+  <xsl:template mode="mode-iso19139" priority="2000" match="gmd:dataQualityInfo/*/gmd:report/gmd:DQ_DomainConsistency/gmd:result/gmd:DQ_ConformanceResult/gmd:specification/gmd:CI_Citation/gmd:title/gmx:Anchor[/root/gui/currTab/text()='inspire_sds']">
+    <xsl:call-template name="render-element">
+      <xsl:with-param name="label" select="/root/gui/strings/category"/>
+      <xsl:with-param name="value" select="@xlink:href"/>
+      <xsl:with-param name="cls" select="local-name()"/>
+      <xsl:with-param name="xpath" select="gn-fn-metadata:getXPath(.)"/>
+      <xsl:with-param name="type" select="'select'"/>
+      <xsl:with-param name="listOfValues" select="/root/gui/schemas/iso19139/codelists/codelist[@name='INSPIRE_category']"/>
```

Comment:
Better to use $codelists variables https://github.com/geonetwork/core-geonetwork/blob/develop/web/src/main/webapp/xslt/common/base-variables-metadata.xsl#L74 (in case one day we move away from Jeeves services layout)